shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts

```diff
@@ -7,7 +7,7 @@ import {
 
 export type JudgeActivityReportCavAndSubmittedCasesRequest = {
   statuses: string[];
-  judges: string[];
+  judges: string[] | undefined;
   pageNumber?: number;
```

Comment:
Could this just be optional?

Reply:
yes! kinda. turns out the two places that are using this type are using two slightly different types, so we separated them into two different ones.

Reply:
Cool! Thanks for the explanation